setup/Swix/Microsoft.FSharp.SDK/Files.swr

```diff
@@ -1,9 +1,9 @@
 use vs
 
-package name=Microsoft.FSharp.SDK.Core
+package name=Microsoft.FSharp.SDK
         version=$(FSharpPackageVersion)
 
-folder "InstallDir:Common7\IDE\CommonExtensions\Microsoft\FSharpSdk"
+folder "InstallDir:MSBuild\15.0\Bin\FSharpSdk"
```

Comment:
@rainersigwald - is this the right sort of convention here?

Reply:
I copied the [Roslyn behavior](https://github.com/dotnet/roslyn/blob/119aec27ff966e55bc28b95ae083aa41d3b8ce17/src/Setup/DevDivVsix/CompilersPackage/Microsoft.CodeAnalysis.Compilers.swr#L12) for this.

Reply:
I would not expect F# to need to be in the MSBuild bin folder. C# is special because we both used to be in the framework. Plus, people often construct relative paths, so changing it can break folks. Did something break with the old path?

Reply:
The original location from the MSI was `C:\Program Files (x86)\Microsoft SDKs\FSharp\...`.  My first round of this PR moved the compiler to `Common7\IDE\CommonExtensions\Microsoft\FSharpSdk`, but that felt wrong for the BuildTools SKU because there is no IDE.  The second round of this PR I moved it to be under MSBuild to copy C#.  Do you have a suggestion for a better place to put it?

Reply:
I would agree with @rainersigwald. I don't this this is an acceptable location, especially given that you're redistributing our old binaries. `Common7\IDE\CommonExtensions\Microsoft\FSharpSdk` seems fine with me, that's where NuGet is and they're required in no-IDE scenarios.